packages/usa-range/src/index.js

```diff
@@ -0,0 +1,59 @@
+const selectOrMatches = require("../../uswds-core/src/js/utils/select-or-matches");
+const behavior = require("../../uswds-core/src/js/utils/behavior");
+
+const { prefix: PREFIX } = require("../../uswds-core/src/js/config");
+
+const RANGE_CLASSNAME = `${PREFIX}-range`;
+const RANGE = `.${RANGE_CLASSNAME}`;
+
+/**
+ * Update range callout for screen readers using the optional data attributes.
+ *
+ * Get optional data attributes, construct and appends aria-valuetext attribute.
+ *
+ * @example
+ *
+ * <input id="usa-range" class="usa-range" type="range" min="0" max="100" step="10" value="20" role="slider" data-text-unit="degrees">
+ *
+ * Callout returns "20 degrees of 100."
+ *
+ * @param {HTMLInputElement} targetRange - The range slider input element
+ */
+const updateCallout = (targetRange) => {
+  const rangeSlider = targetRange;
+  const defaultPrep = "of";
+  const optionalPrep = rangeSlider.dataset.textPreposition;
+  const prep = optionalPrep || defaultPrep;
+  const unit = rangeSlider.dataset.textUnit;
+  const val = rangeSlider.value;
+  const max = rangeSlider.getAttribute("max");
```

Comment:
One more thought: if no `max` attribute is present on the component, the readout will give the value of `null`. For example, "20 of null". Wondering if we could put a default value there (the default value for the max attribute appears to be 100 - [Source MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range#validation)) or a console error.

Reply:
This is a great catch! Added a default value in [b06a84f](https://github.com/uswds/uswds/pull/5472/commits/b06a84f772290ec6f15aa5645256c762199467cc) 👍